WcaOnRails/app/models/competition.rb

```diff
@@ -515,8 +521,12 @@ def has_entry_fee?
     base_entry_fee.nonzero?
   end
 
+  def has_competitor_limit?
+    competitor_limit_flag
```

Comment:
That could be a good option. Is that clearer on the user interface though? At the moment the competitor limit and competition limit reason fields are hidden until the competitor_limit_flag is checked.